object-cache.php

```diff
@@ -12,9 +11,45 @@
 
 // Users with setups where multiple installs share a common wp-config.php or $table_prefix
 // can use this to guarantee uniqueness for the keys generated by this object cache
-if ( !defined( 'WP_CACHE_KEY_SALT' ) ) {
+if ( !defined( 'WP_CACHE_KEY_SALT' ) )
```

Comment:
In general, single line `if` statements should always have the braces; it makes patching down the line much cleaner if you need to expand out to two lines, and it's much more explicit.

*However* with these specific cases, I'd actually prefer the following syntax:
```php
defined( 'WP_CACHE_KEY_SALT' ) or define( 'WP_CACHE_KEY_SALT', DB_NAME );
```

Makes it easier to visually scan.